README.md

```diff
@@ -66,3 +66,61 @@ terraform apply -target module.dns
 
 Create the delegation and ACM validation records as specified by the `dns_delegation_details` and `dns_acm_validation_record` outputs.
 Once that is done you can continue with a full `terraform apply`.
+
+## Authenticating with the AWS CLI
+
+In order to run terraform commands locally you will need to be authenticated to the AWS CLI.
+For security we use [aws-vault](https://github.com/99designs/aws-vault) for securely storing credentials locally.
+
+Prerequisites:
+
+1. Install aws-vault as per the [instructions](https://github.com/99designs/aws-vault#installing)
+2. Have an account in the mhclg AWS account which has permissions to assume the developer role in the Delta-Dev and
+   Delta-Prod accounts
+3. Set up MFA on your account in the mhclg account (required for assuming the role in the other accounts)
+    1. To do this log in to your account in the web console and navigate to IAM
+    2. On the right hand side use the 'Quick links' section to quickly get to the tab 'My security credentials'
+    3. Scroll down to the 'Multi-factor authentication' section and work through the wizard to add an MFA
+4. Generate and AWS Access Key in the mhclg account
+    1. To do this log in to your account in the web console and navigate to IAM
+    2. On the right hand side use the 'Quick links' section to quickly get to the tab 'My security credentials'
+    3. Scroll down to the 'Access keys' section and select 'Create access key'
+    4. Ensure you save your access key somewhere safe - such as in a private folder in Keeper
+
+Setting up AWS Vault:
+
+1. Open your AWS config file in whatever text editor you like
+    - This lives at ~/.aws/config, you can do this from bash by running `nano ~/.aws/config`, this will also create the
+      file if it doesn't exist yet
+2. Add the following contents to the file, filling in your username where needed
+    ```
+   [profile delta]
```

Comment:
On reflection, this profile should maybe be called mhclg since that's the name of the account